### PR TITLE
Update for pypi

### PR DESCRIPTION
**Pull request by YuvalEvron**

### CODE REVIEW

**Comment by YuvalEvron** [comment]
@DoronCohen  - Can you please look at the log rotation area?

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (doron-cohen) [inline .vscode/launch.json:1]: pretty sure all files under `.vscode` should be ignored

**Review by doron-cohen — CHANGES_REQUESTED** [review]
where is the log rotation changes?

**Comment by YuvalEvron** [comment]
> @DoronCohen - Can you please look at the log rotation area?

lib/__init__ line 51

**Comment by YuvalEvron** [comment]
> where is the log rotation changes?

lib/__init__ line 51

**Pull request closed without merging**